packages/core/utils/src/index.js

```diff
@@ -12,6 +12,7 @@ export {default as getCertificate} from './getCertificate';
 export {default as getRootDir} from './getRootDir';
 export {default as isURL} from './is-url';
 export {default as objectHash} from './objectHash';
+export {default as parseCSSImport} from './parseCSSImport';
```

Comment:
I don't think we need to put this in utils yet. We should be able to put this in with the Transformer.

Reply:
I put it in utils so it could be reused for the Sass Transformer in #3353

Reply:
Not sure if we should even keep the special css import syntax, it seems like there's not really a standard for it.

The `~node_module/something` import syntax got introduced by the webpack loader as far as I can tell.

I think the best way forward with this is to put the resolve in a try catch and in catch try to use the default less resolver and if that fails test for special webpack specific import syntax like the `~node_module/something` and throw a clear error for it.

Reply:
Thinking about it for a bit I agree with taking this out, it was never supported for the CSS Asset nor the Stylus Asset, plus the similarity between `~module/style` and `~/module/style` can lead to confusing errors due to typos.

And of course, `~/node_modules/module/style` does the trick just fine.